internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -363,7 +365,6 @@ Resources:
     Metadata:
       'aws:copilot:description': "An IAM role {{- if .PermissionsBoundary}} with permissions boundary {{.PermissionsBoundary}} {{- end}} to update the Route 53 hosted zone"
     Type: AWS::IAM::Role
-    Condition: HasAssociatedDomain
```

Comment:
oo why is this not needed anymore?

Reply:
it's essentially "if alias" logic for now.

Reply:
oh I see: if they don't have `alias` and have a associated domain with the app, they will still access the bucket through the default CF domain?